src/platform/src/common/components/GroupLogo.jsx

```
@@ -2,140 +1,524 @@
+            return source;
+          }
+        } catch {
+          // Try to construct a valid URL if it's a relative path
+          if (source.startsWith('/') || source.startsWith('./')) {
+            try {
+              return new URL(source, window.location.origin).href;
+            } catch {
+              continue;
+            }
+          }
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)